### PR TITLE
Pull to update

### PR DESCRIPTION
**Pull request by salehmoe**
pulling to update my fork

### CODE REVIEW

**Comment by dberzano** [comment]
@salehmoe it is not clear what you were attempting to do, closing this pull request.

@arossi81 this PR should not have been approved, the amount of files changed clearly points to a problem in the PR.

**Comment by arossi81** [comment]
@dberzano : ciao Dario, I am not sure of what this pull request is and why it is related to my approval above, which I think refers to another pull request. Could have been something mixed by mistake?

**Comment by dberzano** [comment]
@arossi81 indeed, you are right, and my apologies for the confusion :-)

It's not a mistake.

So, user @salehmoe wrongly opened a pull request _without syncing his personal repository first_ (we rewrote the full GitHub history on the migration day). Even if he changed "some" files, the PR appears to differ by 376 files (and it has conflicts too).

Your comment appears in the list because it's a _commit_ comment, and that specific commit was included in @salehmoe's pull request. If you remember, this was your comment during the test phase.

Bottom line:

* you should always comment on pull requests, not commits,
* @arossi81, you actually did nothing, but this commit was already commented in the past.

**Comment by arossi81** [comment]
Hi @dberzano ,
yes, clear, I remember what we discussed during the test case.
Cheers,
Andrea
